### DISCUSSION TITLE
Scrape members that comment in a specific post of a specific channel

### DISCUSSION THREAD

**Discussion by hoseini2019** [python-telegram-bot/python-telegram-bot#2924] [Q&A]
Hello, I want to scrape members that comment in a specific post of a specific channel, How do I do that?

**Comment by harshil21** [comment]
Hi,

Unless your bot is part of the linked discussion group since that post, you cannot do this, you would need a userbot. Otherwise, you 
would have to manually keep track of every message in the group and check whether the replied message (update.message.reply_to_message) is your post's.

**Reply by Poolitzer** [reply]
> check whether the replied message is your post's.

or a different comment
